client/src/containers/App/style.scss

```diff
@@ -1,8 +1,41 @@
-#container {
+$use-all: true;
+@import '/../../../../node_modules/animate-sass/animate';
+
+#root {
+  display: block;
   width: 100vw;
   height: 100vh;
 }
 
+#root:after {
+  background-image: url(/images/wallpaper.jpg);
```

Comment:
![image](https://user-images.githubusercontent.com/965232/54039126-21d1bb80-4190-11e9-9432-1ac2b9ffaae4.png)

Reply:
@latch2112 That is a CSS relative path, don't think this can be fixed from "inside" of the IDE. Let me know if you have a solution for this? Otherwise I'll resolved.